nf_core/components/nfcore_component.py

```diff
@@ -86,13 +87,19 @@ def _get_main_nf_tags(self, test_main_nf: str):
                     tags.append(line.strip().split()[1].strip('"'))
         return tags
 
-    def _get_included_components(self, main_nf: str):
+    def _get_included_components(self, main_nf: Union[Path, str]):
         """Collect all included components from the main.nf file."""
         included_components = []
         with open(main_nf, "r") as fh:
             for line in fh:
                 if line.strip().startswith("include"):
-                    included_components.append(line.strip().split()[-1].split(self.org)[-1].split("main")[0].strip("/"))
+                    # get tool/subtool or subworkflow name from include statement, can be in the form
+                    #'../../../modules/nf-core/hisat2/align/main'
+                    #'../bam_sort_stats_samtools/main'
+                    #'../subworkflows/nf-core/bam_sort_stats_samtools/main'
+                    component = line.strip().split()[-1].split(self.org)[-1].split("main")[0].strip("/")
+                    component = component.replace("'../", "subworkflows/")
```

Comment:
Do we have to add "subworkflows/"? 
When the subworkflow is included with the full path the component name won't have the "subworkflows/" part. If we want to specify this, we should add it there too.

Reply:
this command is currently only used to generate tags I think and there we need the `subworkflows/` prefix